master/reference/calicoctl/resources/bgpconfig.md

```diff
@@ -0,0 +1,47 @@
+---
+title: BGP Configuration Resource (BGPConfiguration)
+---
+
+A BGP configuration resource (BGPConfiguration) represents BGP specific configuration options for the cluster.
```

Comment:
This might also be a good place to describe the different types of BGP settings they can configure via this resource. I am not sure, but it seems like there are host-specific settings and global defaults? That might be something pretty interesting to discuss here IMO

Reply:
yeah, I just went with the other resource def style to so it's the same style for all the resources. As per explaining the BGP configurations, there's only 3 configs and they are described in the Spec section as well as there's a yaml example, so it might be a bit redundant to put them in the intro. 
As far as node-specific vs host-specific vs default, there is only node-specific and global (default) which is explained in the Metadata section. Wording might not be the best so I'm open to suggestion there